### ISSUE TITLE
TVDB4 - extra season appearing

### ISSUE THREAD

**Issue by sven-7** [ZeroQI/Absolute-Series-Scanner#118]
I've added Boruto, but when adding it to the library it pops up with a 4th season.  Where in tvdb4.mapping is listed for season 3 as `03|025|025|Field Trip Arc (Ongoing)`, I have a season 4 appearing with season 3 only being one episode long, the 25th.  The metadata is appearing correctly, even with it in season 4, it's just how the scanner has split it up.

On the beta scanner, btw. 

```
Library: 'no valid X-Plex-Token.id', root: 'E:\Series', path: 'Boruto Naruto Next Generations', dirs: '0', subdirs: '[]', files: '31', Scan date: 2017-10-27 00:14:29
=============================================================================================================================================================
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 1 - Uzumaki Boruto!!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 2 - The Hokage's Son!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 3 - Metal Lee Goes Wild!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 4 - A Ninjutsu Battle of the Sexes!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 5 - The Mysterious Transfer Student!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 6 - The Final Lesson!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 7 - Love and Potato Chips!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 8 - The Dream's Revelation!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 9 - Proof of Oneself!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 10 - The Ghost Incident The Investigation Begins!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 11 - The Shadow of The Mastermind.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 12 - Boruto and Mitsuki.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 13 - The Demon Beast Appears!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 14 - The Path That Boruto Can See.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 15 - A New Path.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 16 - Crisis The Threat of Failing!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 17 - Run, Sarada!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 18 - A Day in The Life of The Uzumaki Family.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 19 - Sarada Uchiha.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 20 - The Boy With The Sharingan.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 21 - Sasuke and Sarada.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 22 - Connected Feelings.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 23 - Bonds Come in All Shapes.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 24 - Boruto and Sarada.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 25 - The Turbulent Field Trip!!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 26 - The Mizukage’s Successor.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 27 - A Shinobi Battle of Friendship.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 28 - A Declaration of War.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 29 - The New Seven Ninja Swordsmen!!.mkv
Boruto Naruto Next Generations\Boruto Naruto Next Generations - 30 -   The Sharingan VS The Lighting Blade, Kiba The Fang!.mkv
-------------------------------------------------------------------------------------------------------------------------------------------------------------
Forced ID file: 'tvdb4.id' with id '321285' in series folder
TVDB season mode (4) enabled, tvdb4 mapping url: 'http://rawgit.com/ZeroQI/Absolute-Series-Scanner/master/tvdb4.mapping.xml'
unknown_series_length: False, tvdb_mapping: {19: (2, 19), 20: (2, 20), 21: (2, 21), 22: (2, 22), 23: (2, 23), 24: (2, 24), 25: (3, 25)} (showing changing seasons/episodes only)
-------------------------------------------------------------------------------------------------------------------------------------------------------------
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e001 "Uzumaki Boruto!!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e002 "The Hokage'S Son!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e003 "Metal Lee Goes Wild!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e004 "A Ninjutsu Battle Of The Sexes!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e005 "The Mysterious Transfer Student!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e006 "The Final Lesson!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e007 "Love And Potato Chips!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e008 "The Dream'S Revelation!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e009 "Proof Of Oneself!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e010 "The Ghost Incident The Investigation Begins!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e011 "The Shadow Of The Mastermind"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e012 "Boruto And Mitsuki"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e013 "The Demon Beast Appears!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e014 "The Path That Boruto Can See"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e015 "A New Path"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e016 "Crisis The Threat Of Failing!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e017 "Run, Sarada!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0001e018 "A Day In The Life Of The Uzumaki Family"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e019 (Orig: s01e19) "Sarada Uchiha"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e020 (Orig: s01e20) "The Boy With The Sharingan"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e021 (Orig: s01e21) "Sasuke And Sarada"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e022 (Orig: s01e22) "Connected Feelings"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e023 (Orig: s01e23) "Bonds Come In All Shapes"
"Boruto Naruto Next Generations [tvdb4-321285]" s0002e024 (Orig: s01e24) "Boruto And Sarada"
"Boruto Naruto Next Generations [tvdb4-321285]" s0003e025 (Orig: s01e25) "The Turbulent Field Trip!!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0004e026 (Orig: s01e26) "The Mizukage’S Successor"
"Boruto Naruto Next Generations [tvdb4-321285]" s0004e027 (Orig: s01e27) "A Shinobi Battle Of Friendship"
"Boruto Naruto Next Generations [tvdb4-321285]" s0004e028 (Orig: s01e28) "A Declaration Of War"
"Boruto Naruto Next Generations [tvdb4-321285]" s0004e029 (Orig: s01e29) "The New Seven Ninja Swordsmen!!"
"Boruto Naruto Next Generations [tvdb4-321285]" s0004e030 (Orig: s01e30) "The Sharingan Vs The Lighting Blade, Kiba The Fang!"

```

**Comment by EndOfLine369** [comment]
@sven-7 
Per example in xml, please correct the string to "(unknown length)" instead of "(Ongoing)".
See if that fixes your issue.

```
  <anime tvdbid="[tvdbid]" name="[Series Name]" note="Example entry for ongoing series">
    [    0Num]|[First Ep Num]|[Last Ep Num]|[    0Name]
    [    0Num]|[First Ep Num]|[First Ep Num]|[    0Name] (unknown length)
  </anime>
```

**Comment by ZeroQI** [comment]
Ditto :)

**Comment by sven-7** [comment]
Cool. For some reason, I thought I had seen other things in there before. My bad.

**Comment by sven-7** [comment]
Closed #118.
